lib/ansible/module_utils/basic.py

```diff
@@ -82,6 +82,12 @@
 import datetime
 from itertools import repeat, chain
 
+from ansible.module_utils.pycompat24 import get_exception, literal_eval
+from ansible.module_utils.six import (PY2, PY3, b, binary_type, integer_types,
+        iteritems, text_type, string_types)
+from ansible.module_utils.six.moves import map, reduce
+from ansible.module_utils._text import to_native, to_bytes, to_text
+
```

Comment:
This should not be moved.  They're libraries from the current project so go below the imports for stdlib and third party libraries.